src/main.ts

```diff
@@ -1,44 +1,38 @@
-import Vue from 'vue';
-import VueI18n from 'vue-i18n';
-import i18n from './i18n';
+import { App, createApp } from 'vue';
+import { i18n } from './locales';
 import PsAccounts from '@/components/PsAccounts.vue';
 import AccountPanel from '@/components/panel/AccountPanel.vue';
-import {isOnboardingCompleted} from '@/lib/Helpers';
+import { isOnboardingCompleted } from '@/lib/Helpers';
 
-const version = process.env.VUE_APP_VERSION;
+const version = '5.0.0';
 
-const Components: Record<string, Vue.VueConstructor> = {
+const Components: Record<string, any> = {
   PsAccounts,
-  AccountPanel,
+  AccountPanel
```

Comment:
If this PR is a breaking change, so might as well break more. We can remove ` AccountPanel` here. Maybe we can consider keeping only the standalone version et remove the plugin export.